index.bs

```diff
@@ -413,9 +413,17 @@ The user agent may delay the reporting of layout shifts after a
 until such time as it is known that the event does not begin a fling or scroll
 gesture.
 
-The <a href="https://www.w3.org/TR/uievents/#event-type-mousemove">mousemove</a> and
-<a href="https://www.w3.org/TR/pointerevents/#the-pointermove-event">pointermove</a>
-events are also not excluding inputs.
+All the <a href="https://www.w3.org/TR/pointerevents/#the-pointermove-event">pointermove</a> events after
```

Comment:
Why are pointermoves not excluding inputs but mousemoves behave differently? This does not seem correct to me, I think we want any move after pointerdown which is followed by pointerup and not pointercancel to be excluding input, right? And we can have a note that the user agent can 'buffer' the shifts after a pointerdown before surfacing them, in order to be able to determine the appropriate values for hadRecentInput etc. Does this make sense?